.github/workflows/build.yml

```diff
@@ -11,7 +11,7 @@ permissions:
 
 jobs:
   tests:
-    runs-on: ubuntu-latest
+    runs-on: [self-hosted, acervos]
```

Comment:
All the changes to the `.github/workflows/` files in this PR will need to be removed before this can be merged.  So, there are 6 files you've accidentally changed under that folder which need to be reverted.